.github/workflows/update_skill_json.yml

```diff
@@ -8,3 +7,22 @@ jobs:
     permissions:
       contents: write
     uses: neongeckocom/.github/.github/workflows/skill_update_json_spec.yml@master
```

Comment:
not the right spec anymore

Reply:
The important parts are there, so leaving for now